velox/exec/RowContainer.h

```diff
@@ -310,6 +310,14 @@ class RowContainer {
       vector_size_t resultOffset,
       const VectorPtr& result);
 
+  /// Sets in result all locations with null values in col for rows
+  /// (for numRows number of rows).
+  static void extractNulls(
```

Comment:
It looks like 'result' must be a flat boolean vector. Method documentation needs to say that.

However, is this necessary? Would it be more efficient to populate nulls buffer instead? How are you going to use the 'result'?

Reply:
I started with nulls BufferPtr but then decided to make it consistent with RowContainer::extractProbedFlags API https://github.com/facebookincubator/velox/blob/main/velox/exec/RowContainer.h#L359

BufferPtr might be a bit more efficient, but I think it would be a minimal difference.